examples/drawGeometricFigures/drawGeometricFigures.ino

```diff
@@ -5,22 +5,20 @@
 TroykaOLED myOLED(0x3C);
 
 void setup() {
-  // инициализируем дисплей
-  myOLED.begin();
-  // получаем ширину и высоту дисплея
-  int width = myOLED.getWidth();
-  int height = myOLED.getHeigth();
-  // рисуем две диагонали
-  myOLED.drawLine(0, 0, width - 1, height - 1);
-  myOLED.drawLine(0, height - 1, width - 1, 0);
-  // рисуем окружность в центре дисплея и радиусом 30
-  myOLED.drawCircle(width / 2, height / 2, 30);
-  // рисуем диск (закрашенную окружность) в центре дисплея и радиусом 20
-  myOLED.drawCircle(width / 2, height / 2, 20, true);
+    // инициализируем дисплей
+    myOLED.begin();
+    // получаем ширину и высоту дисплея
+    int width = myOLED.getWidth();
+    int height = myOLED.getHeight();
+    // рисуем две диагонали
+    myOLED.drawLine(0, 0, width - 1, height - 1);
+    myOLED.drawLine(0, height - 1, width - 1, 0);
+    // рисуем окружность в центре дисплея и радиусом 30
+    myOLED.drawCircle(width / 2, height / 2, 30);
+    // рисуем диск (закрашенную окружность) в центре дисплея и радиусом 20
```

Comment:
//draw a disk (filled circle) in the centre of a display and with radius 20

Reply:
I think comments translation we must do in another PR.